src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User result = new UserService().findByEmail(email);
+        if (result == null) {
+            return false;
+        }
+        String em = result.getEmail();
+        String pass = result.getPassword();
+        if (em.equals(email) && pass.equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
You can simplify your code:
```suggestion
        return em.equals(email) && pass.equals(password);
```